docs/tutorials/skweak.ipynb

```diff
@@ -0,0 +1,10139 @@
+    "    for chunk in doc.noun_chunks:\n",
+    "        if chunk[-1].lower_.rstrip(\".\") in {'corp', 'inc', 'ltd', 'llc', 'sa', 'ag'}:\n",
+    "            yield chunk.start, chunk.end, \"COMPANY\"\n",
+    "\n",
+    "# We create the labelling function by giving it a name, and a function to apply\n",
+    "company_detector = skweak.heuristics.FunctionAnnotator(\"company_detector\", company_detector_fun)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "07f530ad",
+   "metadata": {
+    "id": "07f530ad"
+   },
+   "source": [
+    "We can write another example of heuristics for non-commercial organisations by looking for the occurrence of words that are quite typical of public organisations or NGOs:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "5dc59094",
+   "metadata": {
+    "id": "5dc59094"
+   },
+   "outputs": [],
+   "source": [
+    "OTHER_ORG_CUE_WORDS = {\"University\", \"Institute\", \"College\", \"Committee\", \"Party\", \"Agency\",\n",
+    "                       \"Union\", \"Association\", \"Organization\", \"Court\", \"Office\", \"National\"}\n",
+    "def other_org_detector_fun(doc):\n",
+    "    for chunk in doc.noun_chunks:\n",
+    "        if any([tok.text in OTHER_ORG_CUE_WORDS for tok in chunk]):\n",
+    "            yield chunk.start, chunk.end, \"OTHER_ORG\"\n",
+    "\n",
+    "# We create the labelling function\n",
+    "other_org_detector = skweak.heuristics.FunctionAnnotator(\"other_org_detector\", other_org_detector_fun)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d769cd58",
+   "metadata": {
+    "id": "d769cd58"
+   },
+   "source": [
+    "### NER models\n",
+    "\n",
+    "We can also take advantage of machine learning models trained from data of related domains. Here, we will use a spacy model trained on OntoNotes 5.0 to get more named entities."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "1503c939",
+   "metadata": {
+    "id": "1503c939"
+   },
+   "outputs": [],
+   "source": [
+    "ner = skweak.spacy.ModelAnnotator(\"spacy\", \"en_core_web_sm\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "701224ac",
+   "metadata": {
+    "id": "701224ac"
+   },
+   "source": [
+    "Finally, we run our annotators over the documents."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "f70098fa",
+   "metadata": {
+    "id": "f70098fa"
+   },
+   "outputs": [],
+   "source": [
+    "train_docs = list(company_detector.pipe(train_docs))\n",
+    "train_docs = list(other_org_detector.pipe(train_docs))\n",
+    "train_docs = list(ner.pipe(train_docs))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f14cb5ca",
+   "metadata": {
+    "id": "f14cb5ca"
+   },
+   "source": [
+    "## Aggregation\n",
+    "\n",
+    "Once the labelling functions have been applied, we must then aggregate their results, so that we can a single annotation for each document. \n",
```

Comment:
```suggestion
    "Once the labelling functions have been applied, we must then aggregate their results, so that we obtain a single annotation for each document. \n",
```

Reply:
Done. See commit 8ebdfc2 .